packages/nestjs-password/src/services/password-strength.service.ts

```diff
@@ -28,10 +29,17 @@ export class PasswordStrengthService
    * @param password - the plain text password
    * @returns password strength
    */
-  isStrong(password: string): boolean {
-    // Get min password Strength
+  isStrong(
+    password: string,
+    options?: PasswordStrengthOptionsInterface,
+  ): boolean {
+    const { passwordStrength } = options || {};
+
+    // TODO: Should we allow overriding the minimum password strength even if the provided strength is lower than the configured minimum?
     const minStrength =
-      this.settings?.minPasswordStrength || PasswordStrengthEnum.None;
+      passwordStrength ||
+      this.settings?.minPasswordStrength ||
+      PasswordStrengthEnum.None;
```

Comment:
coalesce is safer i think in case strength is explicitly zero? recommend test coverage to be sure.
```suggestion
      passwordStrength ??
      this.settings?.minPasswordStrength ??
      PasswordStrengthEnum.None;
```